src/validator/is-ipv6-addr.ts

```diff
@@ -0,0 +1,72 @@
+import {STRule} from '../rule/rule';
+import {STRuleFn} from '../rule/fn';
+import {STRuleModifiers} from '../rule/modifiers';
+import {STRuleNode} from '../rule/node';
+import {STRuleNodeType} from '../rule/node-type';
+
+export type STOpIsIpv6Addr<CallerType> = () => CallerType;
+
+export const isIpv6Addr = (current: string): boolean => {
+	if (typeof current != 'string') {
+		return false;
+	}
+
+	if (!current.trim()) {
```

Comment:
Similar to my ipv4 feedback, save the result of trim and do the truthy check against that